jessie_update/jessie-heavy-upgrade.sh

```diff
@@ -17,27 +17,46 @@
 ###################################################
 
 ###################################################
+# remove all extraneous packages from Jessie 
+# trying to gain as much space as possible
+# change CleoQc to DexterInd when doing PR
+###################################################
+rm ./jessie_cleanup.sh  # just making sure there isn't one yet
+wget https://raw.githubusercontent.com/CleoQc/Raspbian_For_Robots/master/jessie_update/jessie_cleanup.sh
+chmod +x ./jessie_cleanup.sh
+./jessie_cleanup.sh
+rm ./jessie_cleanup.sh
+
+###################################################
+# give Raspbian its flavor. After this, host will be dex, and pi password will be robots1234
+# Samba will also be installed with password set to robots1234
+###################################################
+rm ./Raspbian_for_Robots_Flavor.sh
+wget https://raw.githubusercontent.com/CleoQc/Raspbian_For_Robots/master/jessie_update/Raspbian_for_Robots_Flavor.sh
+chmod +x ./Raspbian_for_Robots_Flavor.sh
+./Raspbian_for_Robots_Flavor.sh
+rm ./Raspbian_for_Robots_Flavor.sh
+
+###################################################
 ##
-## Take Jessie Lite to Raspbian
+## Take "Jessie on a Diet" to Raspbian
 ##
 ###################################################
 
 sudo apt-get update -y
 sudo apt-get upgrade -y
 
+# is dist-upgrade necessary ? 
 sudo apt-get dist-upgrade -y
+
+# is apache2 necessary? 
 sudo apt-get install apache2 -y
 sudo apt-get install php5 libapache2-mod-php5 -y
 sudo apt-get install raspberrypi-net-mods -y
 sudo apt-get install wpagui -y
 sudo apt-get install scratch -y
 
-sudo apt-get install avahi-autoipd tightvncserver bc python-imaging python-pexpect python-renderpm python-reportlab python-reportlab-accel python-tk python3-tk idle idle-python2.7 idle3 nodejs nodejs-legacy pypy-setuptools pypy-upstream pypy-upstream-dev pypy-upstream-doc blt bluetooth -y
-
-sudo apt-get remove gnome-mplayer yelp deluge claws-mail iceweasel lxmusic audacious transmission-gtk midori netsurf-gtk dillo -y
-
-sudo apt-get autoremove -y
```

Comment:
I was hoping to do the clean and autoremove only once at the end, to save on time. It doesn't create more total space by running it periodically. 
I was only thinking about saving time here, but on the other hand, for modularisation purposes, it might be best to have those two lines whenever packages are removed.